src/hubAuth.ts

```diff
@@ -23,6 +23,38 @@ enum AuthStrategy {
 
 const DEFAULT_INSTANCE_URL = 'https://login.salesforce.com';
 
+/**
+ * Function examines the env var TESTKIT_JWT_KEY to determine if it needs to be
+ * reformatted so when saved to a file the RSA key file contents are formatted
+ * properly.
+ *
+ * Throws an error if function is called and the env var is undefined
+ *
+ * returns a string that complies with RSA private key file format
+ */
+const formatJwtKey = (): string => {
+  if (env.getString('TESTKIT_JWT_KEY')) {
+    let keyLines = env.getString('TESTKIT_JWT_KEY', '').split('\n');
+    if (keyLines.length <= 1) {
+      const footer = '-----END RSA PRIVATE KEY-----';
+      const header = '-----BEGIN RSA PRIVATE KEY-----';
+      // strip out header and footer
+      const newKeyContents = env.getString('TESTKIT_JWT_KEY', '').replace(header, '').replace(footer, '');
+      // check to see if newlines were replaced with spaces
+      keyLines = newKeyContents.trim().split(/\s/);
```

Comment:
On Macos, the newlines are replaced with spaces, so try a split on spaces. Not sure about other platforms.

I may change this to replace spaces with empty string to rid the code of the following conditionals.